security-policies/bundle/compliance/cis_azure/rules/cis_9_3/rule.rego

```diff
@@ -0,0 +1,24 @@
+package compliance.cis_azure.rules.cis_9_3
+
+import data.compliance.lib.common
+import data.compliance.policy.azure.data_adapter
+
+finding = result {
+	# filter
+	data_adapter.is_website_asset
+
+	# set result
+	result := common.generate_result_without_expected(
+		common.calculate_result(is_tls_version("1.2")),
+		data_adapter.resource,
+	)
+}
+
+is_tls_version(version) {
+	data_adapter.site_config.minTlsVersion == version
```

Comment:
what if the minTlsVersion is > 1.2? should we pass?

Reply:
<img width="816" alt="Screenshot 2023-12-04 at 10 32 35" src="https://github.com/elastic/cloudbeat/assets/12246907/4a664b9e-9103-45c5-badb-479674690cf6">